rackcity/utils/asset_utils.py

```diff
@@ -456,16 +461,28 @@ def save_all_field_data(data, asset, change_plan=None):
                     and len(assets_with_asset_number) > 0
                     and assets_with_asset_number[0].id != asset.id
                 ):
-                    return (
+                    return (None,
                         "Asset with asset number '"
                         + str(data[field])
                         + "' already exists."
                     )
             value = data[field]
         else:
             value = data[field]
-        setattr(asset, field, value)
-        try:
+
+        print(field, value)
+        if field is not "id":
+            setattr(asset, field, value)
+
+    try:
+        if create_asset_cp:
+            asset_cp = copy_asset_to_new_asset_cp(asset, change_plan)
```

Comment:
make all the modifications on the original live asset, but dont save it-> copy values to a new assetcp